ci/tcf-tests.yaml

```diff
@@ -15,23 +15,41 @@
 version: '3.5'
 
 services:
-  tcf-tests:
-    image: tcf-ci-build:${ISOLATION_ID}
+  avalon-tests:
+    image: avalon-shell:${ISOLATION_ID}
     build:
       context: .
       args:
         - http_proxy
         - https_proxy
         - no_proxy
-    working_dir: "/project/TrustedComputeFramework/"
+    working_dir: "/project/avalon/"
     entrypoint: "bash -c \"\
-        source scripts/tcs_startup.sh -y -l http://avalon-lmdb:9090 \
-        && sleep 5 \
-        && ./tools/run_tests.sh -l avalon-listener \""
+        ./tools/run_tests.sh -l avalon-listener \""
```

Comment:
in avalon-tests service, you are running tools/run_tests.sh file. This service uses docker/Dockerfile.tcf-dev which compiles the entire Avalon code through rebuild.sh. Why do we need to compile the entire source code? Please do the clean-up of docker/Dockerfile.tcf-dev

Reply:
This is done because we don't have separate dockerfile for shell/client service. I understand that we are copying and compiling whole source code which is not necessary. But i want to avoid make changes to shell service and mix it up with this PR. I could raise a separate PR to address this.

Reply:
I understand, dockerfile for shell/client can be done in seperate PR. But the dockerfile used should be updated (remove unwanted stuff from there). This will make it faster.

Reply:
Ok, i will optimize the dockerfile.